docs/_layouts/default.html

```diff
@@ -27,7 +27,7 @@
 		<script src="{{ site.baseurl }}/assets/js/vendor/moment-with-locales.min.js"></script>
 		<script src="{{ site.baseurl }}/assets/js/vendor/Chart-2.7.1.min.js"></script>
 		<script src="{{ site.baseurl }}/assets/js/vendor/spin-2.3.2.min.js"></script>
-		<script src="{{ site.baseurl }}/assets/js/charts.js?version=1ff0187"></script>
+		<script src="{{ site.baseurl }}/assets/js/charts.js?version=e7e9c5a"></script>
```

Comment:
Is there a way to generate a random hash or a hash over a file with Jekyll (via GitHub pages)?
/cc @parkr 😄

Reply:
`site.github.build_revision` would work on Pages. https://help.github.com/articles/repository-metadata-on-github-pages/

Reply:
@parkr: Thanks for the pointer! This would generate new `?version=` hashes for each commit, right? Do you know of a way to, for example, get a hash based on the actual file content? In this way, we wouldn’t force client or server cache flushes with every single commit. (I’m just asking, your solution would solve our problem pretty well already!)

Reply:
No hashes on a per-file basis are generated, so that’s the best solution that will achieve cache misses based on what’s in Pages.

Reply:
Okay, then this is the way to go. Thanks, @parkr!